resource/specs/testnet.toml

```diff
@@ -75,7 +75,6 @@ lock.hash_type = "type"
 [params]
 epoch_reward = 1_250_000_00000000
 secondary_epoch_reward = 600_000_00000000
-max_block_cycles = 10_000_000_000
```

Comment:
Leave a comment here, that testnet uses the default `MAX_BLOCK_CYCLES` in spec/src/consensus.rs

Reply:
I think @quake 's suggestion is good, and we should remove the entire `params` section from `testnet.toml` until we actually need to config these fields with non-default values.